src/install.rs

```diff
@@ -448,7 +448,7 @@ async fn prepare_build<'a>(
         false
     };
 
-    if !config.skip_review {
+    if config.skip_review == "ask" {
```

Comment:
This should be != never. But that's all I see wrong now.

Reply:
If it's `!= never` then it will also be true for `always` but `always` shouldn't ask for user input.

Reply:
The double negative keeps getting me. != always in that cause. Basically the prompt should ask if you want to review you don't skip the review process.

Reply:
I'm not quite sure i understand you.
That line should be `== "ask"` because only `ask` should prompt the user for input, both `always` and `never` should not prompt the user for input.

Reply:
Look at the else.

Reply:
Can you elaborate? I see nothing wrong.

Reply:
All the if does is change the prompt message. If there's any chance to review (skip = ask or never) then the prompt should ask to continue to review.

Reply:
Changing the prompt message is the same behavior as before. 

If the user sets `SkipReview` to `never` then they should not be prompted (otherwise `never` has the same the same effect as `ask`),  they should only be prompted for input when `SkipReview` is set to `ask`. That is the entire idea of implementing `always` and `never` for `SkipReview` in the first place, if the user always or never reviews diffs then they should not have to be prompted for input every time.

Reply:
The prompt is unrelated to review. It simply asks if you want to continue to the next step. It's an if-else that just changes the message. You're unconditionally prompted. The only issue with this PR the prompt message is wrong when skipreview = never.

Reply:
I thought there was an actual prompt to ask if you want to review but I guess I may have confused that with yay. In that case you'd be free to add one but given what you want is to always review I doubt you'd want that.

Reply:
Ok, i think i see what you mean, please have a look at the new commit.

Reply:
To summarize:
Regardless of `SkipReview`s value the same prompt after displaying conflicts, etc, asking the user to continue with the installation.
Then, if `SkipReview` is set to `ask`, the user is prompted for input whether or not the review the diffs.
If the user answered yes or if `SkipReview` is set to `never` then the review is shown.
If `SkipReview` is set to `always` the user is not prompted and the diffs are not shown.

Reply:
I added the `ask_review` function as i think if `SkipReview` is set to ask then `--no-confirm` should not have any effect on the prompt.

Reply:
So what you really want is just the ability to use --noconfirm and also review?

Reply:
Essentialy yes, that is the behavior i want. See #254.

Reply:
#254 doesn't ask for the behaviour you're describing. If you just want --noconfirm with pause to confirm the review I think this isn't really the write aproch. --noconfirm needs to not prompt for the sake of scripting. I think a dedicated flag would be better than chucking it in with skip review.

Reply:
Hmm, maybe i wasn't clear. Like i said, that's the behavior i want but it's not the use case i want. I just meant that using `--noconfirm` and getting to review, would produce the same result as what i wanted for the review process. But using `--noconfirm` would also affect the rest of the entire installation process.
This is unrelated to `--noconfirm`, that was just something i had to workaround because how `ask` (the function) was implemented. 
Personally i don't use `--no-confirm`, i just don't want to be prompted whether or not review everytime, because i always do review.

Reply:
Okay, but as is, Paru does not ask if you want to review it does automatically.

Reply:
Are you saying that paru does not ask the user if they want to review? Because that is not the behavior that i'm seeing when installing packages
```
# paru tealdeer
1 aur/tealdeer 1.4.1-5 [+42 ~3.11] [Installed]
    A fast tldr client in Rust.
2 aur/tealdeer-git r255.30b7c5f-1 [+10 ~0.00]
    A fast tldr client in Rust.
3 aur/tealdeer-bin 1.4.1-1 [+6 ~0.61]
    A fast tldr client in Rust. Static binaries from GitHub.
:: Packages to install (eg: 1 2 3, 1-3):
:: 2
:: Resolving dependencies...
:: Calculating conflicts...
:: Calculating inner conflicts...

:: Conflicts found:
    tealdeer-git: tealdeer (tldr)  tealdeer (tldr)

:: Conflicting packages will have to be confirmed manually

Aur (1) tealdeer-git-r255.30b7c5f-1

:: Proceed to review? [Y/n]:
```

Reply:
Yes, paru tell you what it's going to install then asks if you're okay with that and want to continue to the next step. It is nothing to do with review.

Reply:
Not to mention after this prompt pkgbuilds are downloaded. So the prompt is also there to stop you from just downloading pkg builds accidentally.

Reply:
Ah, yes i see what you mean now. I guess this pr is redudant then. Should i close it?